src/listview.cpp

```diff
@@ -874,36 +874,52 @@ QString ListView::refNameAt(const QPoint &pos)
 	return QString();
 }
 
+/*
+ * Return the device pixel ratio
+ */
+const qreal ListViewDelegate::dpr(void) const {
```

Comment:
This does not need to be a private method on the class. It can very well be a local function and releave a tiny bit of the burden from the compiler.